### PR TITLE
fix: Removed europe region and converted jitter values from s to ms

### PR DESCRIPTION
**Pull request by monkCommits**
# Description

Removed the region Europe as it is not needed as of now.
Converted jitter values from s to ms.

_List which issues are fixed by this PR. You must list at least one issue._
- Europe listed in the Regions.
- Jitter values appearing in s

### Pre-launch Checklist

- [x] The [Documentation] is updated accordingly, or this PR doesn't require it.
- [x] I updated/added relevant documentation.
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I added new tests to check the change I am making, or this PR is test-exempt.
- [x] All existing and new tests are passing.

### Merging:
- Squash merge to dev
- Merge commit to publish-alpha and main

[Documentation]: https://www.100ms.live/docs

### CODE REVIEW

**Review by KaustubhKumar05 — DISMISSED** [review]

File: packages/roomkit-react/src/Diagnostics/ConnectivityTest.tsx [hunk 1]
```diff
@@ -164,7 +163,6 @@ const AudioStats = ({ stats }: { stats: DiagnosticsRTCStats | undefined }) => {
           <DetailedInfo title="Bitrate Sent" value={formatBytes(stats.bitrateSent, 'b/s')} />
           <DetailedInfo title="Bitrate Received" value={formatBytes(stats.bitrateReceived, 'b/s')} />
           <DetailedInfo title="Round Trip Time" value={`${stats.roundTripTime} ms`} />
-          <DetailedInfo title="Jitter" value={`${stats.jitter} s`} />
```
Comment (ygit) [inline packages/roomkit-react/src/Diagnostics/ConnectivityTest.tsx:167]: @monkCommits why is it removed here?

File: packages/roomkit-react/src/Diagnostics/ConnectivityTest.tsx [hunk 2]
```diff
@@ -183,7 +181,6 @@ const VideoStats = ({ stats }: { stats: DiagnosticsRTCStats | undefined }) => {
...
```
Comment (ygit) [inline packages/roomkit-react/src/Diagnostics/ConnectivityTest.tsx:186]: Similarly, @monkCommits why is it removed?

**Review by ygit — CHANGES_REQUESTED** [review]

**Review by ygit — APPROVED** [review]
LGTM

File: packages/roomkit-react/src/Diagnostics/ConnectivityTest.tsx [hunk 3]
```diff
@@ -163,6 +163,7 @@ const AudioStats = ({ stats }: { stats: DiagnosticsRTCStats | undefined }) => {
           <DetailedInfo title="Bitrate Sent" value={formatBytes(stats.bitrateSent, 'b/s')} />
           <DetailedInfo title="Bitrate Received" value={formatBytes(stats.bitrateReceived, 'b/s')} />
           <DetailedInfo title="Round Trip Time" value={`${stats.roundTripTime} ms`} />
+          <DetailedInfo title="Jitter" value={`${stats.jitter} s`} />
```
Comment (KaustubhKumar05) [inline packages/roomkit-react/src/Diagnostics/ConnectivityTest.tsx:166]: This should also be in ms

File: packages/roomkit-react/src/Diagnostics/ConnectivityTest.tsx [hunk 4]
```diff
@@ -181,6 +182,7 @@ const VideoStats = ({ stats }: { stats: DiagnosticsRTCStats | undefined }) => {
...
```
Comment (KaustubhKumar05) [inline packages/roomkit-react/src/Diagnostics/ConnectivityTest.tsx:185]: Convert this too, we need it to be consistent and ms everywhere

Comment (monkCommits) [reply packages/roomkit-react/src/Diagnostics/ConnectivityTest.tsx]: @KaustubhKumar05 updated

**Review by KaustubhKumar05 — APPROVED** [review]

**Comment by KaustubhKumar05** [comment]
Let's merge this after the release

**Review by ygit — APPROVED** [review]
LGTM

**Pull request merged**